### ISSUE TITLE
Multiple UrhoSurface in one WinForms-application

### ISSUE THREAD

**Issue by alexsigmund** [xamarin/urho#344]
I have a WinForms application in which i have a UrhoSurface and a dialog that has also an UrhoSurface.
The Problem is that the UrhoSurface of the Dialog, when i open the dialog, is closing my Application that is running on the UrhoSurface on my MainForm.
I need that they can coexist in parallel. 

It would be also a possible option to pause the Application on the UrhoSurface that is on the MainForm, and then resume it when the dialog gets closed.

**Comment by EgorBo** [comment]
Yeah unfortunately urhosharp doesn't support such scenario 🙁 You have to re-create surface each time you close that dialog
